src/style/components/Login/_loginForm.scss

```diff
@@ -1,7 +1,6 @@
 .loginPage {
   .welcomeTitle {
     text-align: center;
-    margin-top: 15%;
```

Comment:
This was creating weird issues when resizing - should probably look at this page in detail